src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -330,6 +341,40 @@ export default {
 			}
 		},
 
+		/**
+		 * WebImage handlers
+		 */
+		async selectSocialAvatar() {
+
+			console.debug(this.contact)
+
+			const apiUrl = generateUrl('apps/contacts/api/v1/social/')
+			const addressbookId = this.contact.addressbook.id
+			const contactId = this.contact.uid
+			const imageUrl = apiUrl + 'avatar/' + addressbookId + '/' + contactId
+
+			if (!this.loading) {
+
+				this.loading = true
+				try {
+					const { get } = await axios()
```

Comment:
Regarding this, I don't know what I ddid on this file, but the axio simport is not correct!
That's why you have csfr issues :thinking: 

You can just do axios.get(), and you need to replace the import by `import axios from '@nextcloud/axios'` Sorry abou tthat, I don't get my code :thinking:

Reply:
Ah, perfect - that solved my CSRF issue!

Reply:
@nextcloud/axios is a library of ours that just use axios but inject the proper csfr token on each request made :)

Reply:
I will change it for the two routines in here, maybe you can check if its also important for the SettingsImportContacts.vue ?

Reply:
Will do later :)